app/assets/stylesheets/application.css

```diff
@@ -627,66 +676,42 @@ li .comment_folder.comment_folder_inline.force_inline {
 li .comment_folder:before {
 	content: "[−]";
 }
-li .comment_folder_button:checked ~ .comment .comment_folder:before {
+.comment_folder_button:checked ~ .comment_gutter .comment_folder:before,
+*:has(> .comment_folder_button:checked) > .comment .comment_folder:before {
 	content: "[+]";
 }
-li .comment_folder_button:checked ~ div.comment div.comment_text {
- 	display: none;
-}
-li .comment_folder_button:checked ~ div.comment div.voters {
-	visibility: hidden;
-	margin-bottom: -15px;
-}
-li .comment_folder_button:checked ~ ol.comments ol,
-li .comment_folder_button:checked ~ ol.comments div.comment,
-li .comment_folder_button:checked ~ ol.comments li {
- 	display: none;
+#inside li .comment_folder_button:checked ~ .comment div.comment_text,
+#inside li .comment_folder_button:checked ~ .comment .comment_gutter > *:not(.comment_folder),
+#inside li .comment_folder_button:checked ~ ol.comments {
+	display: none;
 }
 
 li.comments_subtree {
-	position: relative;
+	grid-column: begin / end;
+	display: grid;
+	grid-template-columns: subgrid;
 }
 
 li .comment_parent_tree_line {
-	position: absolute;
-	left: 30px;
+	flex: 0.5rem 1;
+	width: 0;
 	border-left: 1px dotted var(--color-fg-shape);
-	top: 30px;
-	bottom: 0;
 }
 
-li .comment_parent_tree_line.score_shown {
-	top: 44px;
-}
-li .comment_parent_tree_line.can_flag {
-	top: 56px;
-}
 /* hide for comments that don't have children */
 li:not(:has(> .comments > li)) > .comment > .comment_parent_tree_line {
```

Comment:
This selector has broken. This is a small feature of tree lines and my comment wasn't clear enough (so that should be rewritten, too!). If a comment is the last at its level of reply (or in other words, has no siblings after it), it shouldn't have a line, even if it has replies. The lines are for making siblings clear when the parent might have scrolled off screen, it's sort of assumed that the indent is enough for direct replies.